src/main/java/org/apache/datasketches/count/CountMinSketch.java

```diff
@@ -22,8 +22,8 @@
 import org.apache.datasketches.common.Family;
 import org.apache.datasketches.common.SketchesArgumentException;
 import org.apache.datasketches.common.SketchesException;
+import org.apache.datasketches.common.Util;
 import org.apache.datasketches.hash.MurmurHash3;
-import org.apache.datasketches.tuple.Util;
```

Comment:
This reference to /tuple.Util is only used for the computeSeedHash() in lines 364, 402 and 404.  Because this method and the companion method checkSeedHashes() are used in other sketches and not just the Tuple Sketch, these two functions have been moved to /datasketches.common.Util. in the current main. All you need to do is change this import to .../datasketches.common.Util.